unlock-app/src/__tests__/components/interface/user-account/KeyPurchaseConfirmation.test.tsx

```diff
@@ -130,19 +134,24 @@ describe('KeyPurchaseConfirmation', () => {
         emailAddress,
         address,
         lock,
+        price,
       })
     })
 
     it('provides defaults when values are not available', () => {
       expect.assertions(1)
       const state = {
         account: {},
-        cart: {},
+        cart: {
+          price,
```

Comment:
Price shouldn't ever be set if there's no lock in the cart, so you can probably leave it out of the initial state here (and then it should be undefined in the expect)